src/csps/src/CMakeLists.txt

```diff
@@ -1,12 +1,9 @@
 install(FILES csps.py threespace_api.py threespace_utils.py win32_threespace_utils.py DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
I am wondering if there is another command in cmake to run a process that is different from the "install" command i want to reserve install commands